proto/api/v1/administration_events_service.proto

```diff
@@ -0,0 +1,145 @@
+    // to uniquely identify the object. For example, a resource name is a valid resource ID.
+    string resource_id                         = 8;
+
+    // Occurrences associated with the event. When events may occur multiple times, the
+    // occurrences track the amount.
+    int64 num_occurrences                      = 9;
+
+    // Specifies the time when the event has last occurred.
+    google.protobuf.Timestamp last_occurred_at = 10;
+
+    // Specifies the time when the event has been created.
+    google.protobuf.Timestamp created_at       = 11;
+}
+
+// AdministrationEventType exposes the different types of events.
+enum AdministrationEventType {
+    ADMINISTRATION_EVENT_TYPE_UNKNOWN     = 0;
+    ADMINISTRATION_EVENT_TYPE_GENERIC     = 1;
+    ADMINISTRATION_EVENT_TYPE_LOG_MESSAGE = 2;
+}
+
+// AdministrationEventLevel exposes the different levels of events.
+enum AdministrationEventLevel {
+    ADMINISTRATION_EVENT_LEVEL_UNKNOWN = 0;
+    ADMINISTRATION_EVENT_LEVEL_INFO    = 1;
+    ADMINISTRATION_EVENT_LEVEL_SUCCESS = 2;
+    ADMINISTRATION_EVENT_LEVEL_WARNING = 3;
+    ADMINISTRATION_EVENT_LEVEL_ERROR   = 4;
+}
+
+message AdministrationEventsFilter {
+    // Matches events with last_occurred_at after a specific timestamp, i.e. the lower boundary.
+    google.protobuf.Timestamp from  = 1;
+
+    // Matches events with last_occurred_at before a specific timestamp, i.e. the upper boundary.
+    google.protobuf.Timestamp until = 2;
+
+    // Matches events from a specific domain.
+    string domain                   = 3;
+
+    // Matches events associated with a specific resource type.
+    string resource_type            = 4;
+
+    // Matches events based on their type.
+    AdministrationEventType type    = 5;
+
+    // Matches events based on their level.
+    AdministrationEventLevel level  = 6;
+}
+
+message CountAdministrationEventsRequest {
+    // For filtering the events based on the requested fields.
+    AdministrationEventsFilter filter = 1;
+}
+
+message CountAdministrationEventsResponse {
+    // The total number of events after filtering and deduplication.
+    int64 count = 1;
+}
+
+message GetAdministrationEventResponse {
+    AdministrationEvent event = 1;
+}
+
+message ListAdministrationEventsRequest {
+    // For dividing the events response into chunks.
+    Pagination pagination             = 1;
+
+    // For filtering the events based on the requested fields.
+    AdministrationEventsFilter filter = 2;
+}
+
+message ListAdministrationEventsResponse {
+    repeated AdministrationEvent events = 1;
+}
+
+service AdministrationEventService {
+    // CountAdministrationEvents returns the number of events after filtered by requested fields.
+    rpc CountAdministrationEvents(CountAdministrationEventsRequest) returns (CountAdministrationEventsResponse) {
+        option (google.api.http) = {
+            get: "/v1/administration/eventscount"
```

Comment:
So we're not following Mandar's suggestion to route this to `/v1/counts/administration/events`?

Reply:
Hmm, to be honest, I haven't seen / followed that comment since it was resolved already.

If I understand the comment correctly, I'll just use the API path `/v1/count/<resource` for now

cc: @md2119 for visibility since the other thread / comment has been resolved.